### PR TITLE
Update favicons

### PR DESCRIPTION
**Pull request by reducetoflow**
Updated favicons to new logo, added background colour for iOS, add new tiles for Windows.

### CODE REVIEW

File: public/static/favicons/browserconfig.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-      <wide310x150logo src="/static/favicons/mstile-310x150.png"/>
-      <TileColor>#da532c</TileColor>
-    </tile>
-  </msapplication>
-</browserconfig>
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+  <msapplication>
+    <tile>
+      <square70x70logo src="/mstile-70x70.png"/>
```
Comment (danni) [inline public/static/favicons/browserconfig.xml]: These paths look strange... should be "/static/favicons/"

File: public/static/favicons/manifest.json [hunk 2]
```diff
@@ -2,28 +2,40 @@
...
 		},
 		{
-			"src": "\/static\/favicons\/android-chrome-96x96.png",
+			"src": "\/android-chrome-96x96.png",
 			"sizes": "96x96",
 			"type": "image\/png",
 			"density": "2.0"
+		},
+		{
+			"src": "\/android-chrome-144x144.png",
```
Comment (danni) [inline public/static/favicons/manifest.json]: And here.

**Comment by danni** [comment]
And the Apple icons in `src/server/HtmlDocument.js` please :)

**Comment by reducetoflow** [comment]
Should be corrected now, thanks for catching those.

**Comment by danni** [comment]
Can you rebase this flat without the merges?

**Comment by danni** [comment]
You've also introduced a bunch of trailing whitespace.

**Comment by danni** [comment]
(Which is why Travis is failing).

**Comment by reducetoflow** [comment]
Sorry, should be all fixed now

**Comment by danni** [comment]
Awesome, but can you rebase/merge it into a single commit please?

**Pull request merged**